src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -37,20 +42,30 @@ public void configure() {
 
     climberConfig.SoftwareLimitSwitch.ReverseSoftLimitEnable = true;
     climberConfig.SoftwareLimitSwitch.ReverseSoftLimitThreshold = climberPref.climberMotorReverseLimit.getValue();
+
+    climberConfig.MotorOutput.NeutralMode = constClimber.PITCH_NEUTRAL_MODE_VALUE;
     climberMotor.getConfigurator().apply(climberConfig);
+
+    climberMotor.setControl(climberMotorReq.withOutput(12));
   }
 
   public void setClimberMotorSpeed(double motorSpeed) {
 
     climberMotor.set(motorSpeed);
   }
 
+  public double getPosition() {
+    return climberMotor.getPosition().getValue();
+  }
+
   public void setNeutralOutput() {
     climberMotor.setControl(new NeutralOut());
   }
 
   @Override
   public void periodic() {
+    SmartDashboard.putNumber("Climber position(Rotations)", getPosition());
+    SmartDashboard.putString("climber motor encoder", "DutyCycle");
```

Comment:
What is this line doing?

Reply:
nu uh

Reply:
we where trying to show the control type should we have this?